### ISSUE TITLE
Im receiving this error. what does it mean?

### ISSUE THREAD

**Issue by hyuseeee** [dante4rt/getgrass-bot#37]
Received PONG: {"id":"Tj0","action":"PONG"}
Received message: {"id":"0UM","action":"PONG"}
Received PONG: {"id":"0UM","action":"PONG"}
Received message: {"id":"VqX","action":"PONG"}
Received PONG: {"id":"VqX","action":"PONG"}
Received message: {"id":"GMR","action":"PONG"}
Received PONG: {"id":"GMR","action":"PONG"}
Received message: {"id":"USj","action":"PONG"}
Received PONG: {"id":"USj","action":"PONG"}

it appeared on the dashboard but with zero point

**Comment by eboot** [comment]
same, need update bot please

**Comment by zamiguinbra** [comment]
my bot is working normally, I'm using residential scrape proxy, try using it without https and socks5

**Comment by zamiguinbra** [comment]
Yes, mine only has 20 active connections, the other 30 are at zero. the bot needs an update
